src/kadmin/ktutil/ktutil.c

```diff
@@ -149,6 +150,7 @@ void ktutil_add_entry(argc, argv)
         }
         if ((strlen(argv[i]) == 2) && !strncmp(argv[i], "-k", 2)) {
             kvno = (krb5_kvno) atoi(argv[++i]);
+            use_kvno++;
```

Comment:
@mkazia Thanks for attending to this as well. It's useful to be able to use kvno 0.